### PR TITLE
Fixes #19113: The logrotate conf file still uses legacy service reloading

### PR DESCRIPTION
**Pull request by Fdall**
https://issues.rudder.io/issues/19113

### CODE REVIEW

File: techniques/system/server-roles/1.0/rudder-logrotate [hunk 1]
```diff
@@ -59,7 +59,7 @@
         create ${root_logrotate_check.syslog_file_mode} ${root_logrotate_check.syslog_user} ${root_logrotate_check.syslog_group}
         delaycompress
         postrotate
-          ${root_logrotate_check.service_restart_utility} ${root_logrotate_check.syslog_service_name} ${root_logrotate_check.syslog_rotation_method} > /dev/null || true
+          ${paths.path[systemctl]} ${root_logrotate_check.syslog_rotation_method} ${root_logrotate_check.syslog_service_name} > /dev/null || true
```
Comment (amousset) [inline techniques/system/server-roles/1.0/rudder-logrotate:62]: does `systemctl rotate` exist?

Comment (ncharles) [reply techniques/system/server-roles/1.0/rudder-logrotate:62]: also on sles12sp1, is there a systemctl ?

Comment (Fdall) [reply techniques/system/server-roles/1.0/rudder-logrotate:62]: You are right, this won't work on debian based system where we are curently calling the init rotate command which send a HUP signal to syslog.
I will close this PR as it is not as straightforward as expected.

**Comment by Fdall** [comment]
Closing this pr since it may break things and only bring non yet mandatory changes.

File: techniques/system/server-roles/1.0/rudder-logrotate (hunk 1 above)
Comment (amousset) [reply techniques/system/server-roles/1.0/rudder-logrotate:62]: SIGHUP is generally "systemctl reload"

**Pull request closed without merging**